cmd/cloud/deployment.go

```diff
@@ -134,7 +134,7 @@ func newDeploymentCreateCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&clusterID, "cluster-id", "c", "", "Cluster to create the Deployment in")
 	cmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "v", "", "Runtime version for the Deployment")
 	cmd.Flags().StringVarP(&dagDeploy, "dag-deploy", "", "disable", "Enables DAG-only deploys for the deployment")
-	cmd.Flags().StringVarP(&executor, "executor", "e", deployment.CeleryExecutor, "The executor to use for the deployment. Possible values can be CeleryExecutor or KubernetesExecutor.")
+	cmd.Flags().StringVarP(&executor, "executor", "e", "", "The executor to use for the deployment. Possible values can be CeleryExecutor or KubernetesExecutor.")
```

Comment:
nit: we could retain the default for create deployment command, the advantage being it will also show up in the help command. If so you would need to separate variable for executor flag in create & update deployment commands

Reply:
I like having one `executor` var for both create and update to make reading easier. Its ok to explicitly set its default value for create.